src/main/jhipster/src/main/webapp/app/germplasm-details/audit/attributes/germplasm-attribute-audit.route.ts

```diff
@@ -0,0 +1,10 @@
+import { Routes } from '@angular/router';
+import { GermplasmAttributeAuditPopupComponent } from './germplasm-attribute-audit.component';
+
+export const germplasmAttributeAuditRoutes: Routes = [
+    {
+        path: 'germplasm/:gid/attribute/:attributeId/audit-dialog',
```

Comment:
Not mandatory but, if you can rename it to `/attributes/`, we'll be following the same RESTful URL pattern as the API, in which we use plurals for resources (same for `/name/`)

Reply:
Done!